mist-core/src/main/java/edu/snu/mist/core/driver/MistDriver.java

```diff
@@ -164,24 +244,107 @@ public void onNext(final ActiveContext activeContext) {
           .set(NameResolverConfiguration.NAME_SERVICE_PORT, nameServer.getPort())
           .set(NameResolverConfiguration.NAME_SERVER_HOSTNAME, localAddressProvider.getLocalAddress())
           .build();
-      // Task configuration
-      final Configuration taskConfiguration = TaskConfiguration.CONF
-          .set(TaskConfiguration.IDENTIFIER, taskId)
-          .set(TaskConfiguration.TASK, MistTask.class)
-          .set(TaskConfiguration.ON_CLOSE, MistTask.TaskCloseHandler.class)
-          .build();
-      // submit a task
-      activeContext.submitTask(
-          Configurations.merge(nameResolverConf, taskConfiguration, mistTaskConfigs.getConfiguration()));
+      // Configuration for NCS of mist task.
```

Comment:
NCS?

Reply:
Just added it in case when it becomes necessary.